colour/models/icacb.py

```diff
@@ -0,0 +1,157 @@
+# -*- coding: utf-8 -*-
+"""
+ICaCb Colourspace
+=================
+
+Defines the *ICaCb* colourspace transformations:
+
+-   :func:`colour.XYZ_to_ICaCb`
+-   :func:`colour.ICaCb_to_XYZ`
+
+References
+----------
+-   :cite:`Fröhlich2017` : Fröhlich, J. (2020).Encoding High Dynamic Range and
+    Wide Color Gamut Imagery. http://dx.doi.org/10.18419/opus-9664
+"""
+
+from colour.utilities.common import to_domain_1
+from colour.models.rgb.transfer_functions import (eotf_ST2084,
+                                                  eotf_inverse_ST2084)
+import numpy as np
+
+__author__ = 'Colour Developers'
+__copyright__ = 'Copyright (C) 2013-2021 - Colour Developers'
+__license__ = 'New BSD License - https://opensource.org/licenses/BSD-3-Clause'
+__maintainer__ = 'Colour Developers'
+__email__ = 'colour-developers@colour-science.org'
+__status__ = 'Production'
+
+__all__ = [
+    'MATRIX_XYZ_TO_ICACB_1',
+    'MATRIX_XYZ_TO_ICACB_2',
+    'XYZ_to_ICaCb',
+    'ICaCb_to_XYZ',
+]
+
+MATRIX_XYZ_TO_ICACB_1 = np.array([
```

Comment:
*ICACB* like *IPT* and Co. has two matrices the first, this one converts from *CIE XYZ* tristimulus values to LMS cone space. The second one from non-linear LMS cone space to the final colourspace representation. So we should probably rename this matrix as follows for consistency: 

```python
MATRIX_ICACB_XYZ_TO_LMS = np.array([
    [0.37613, 0.70431, -0.05675],
    [-0.21649, 1.14744, 0.05356],
    [0.02567, 0.16713, 0.74235],
])
"""
*CIE XYZ* tristimulus values to normalised cone responses matrix.

MATRIX_ICACB_XYZ_TO_LMS : array_like, (3, 3)
"""
```

We should also precompute the inverse, i.e.,

```python
MATRIX_ICACB_LMS_TO_XYZ = np.linalg.inv(MATRIX_ICACB_XYZ_TO_LMS)

"""
Normalised cone responses to *CIE XYZ* tristimulus values matrix.

MATRIX_ICACB_LMS_TO_XYZ : array_like, (3, 3)
"""
```